src/plugins/chart_expressions/expression_metric/public/components/with_auto_scale.tsx

```diff
@@ -6,13 +6,14 @@
  * Side Public License, v 1.
  */
 
-import React, { useRef, useEffect, useState, ComponentType, useMemo } from 'react';
+import React, { useRef, useEffect, useState, ComponentType, useMemo, CSSProperties } from 'react';
 import { throttle } from 'lodash';
 import { useResizeObserver } from '@elastic/eui';
 import { autoScaleWrapperStyle } from './with_auto_scale.styles';
 
 interface AutoScaleParams {
```

Comment:
I would prefer this `params` to be flat according to the wrapped component. 

```ts
interface Props {
     minScale?: number;
     scaleContainerStyles?: CSSProperties;
     ...
}
```

Reply:
I prefer that we have config object which includes all params related to auto scale

Reply:
@VladLasitsa, ok, as you wish) that is not critical)